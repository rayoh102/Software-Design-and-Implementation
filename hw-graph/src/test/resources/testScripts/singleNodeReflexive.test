# Tests reflexive edge addition with a single node

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# Add reflexive edge
AddEdge graph1 n1 n1 e1

# See if edge is reflexive
ListChildren graph1 n1
