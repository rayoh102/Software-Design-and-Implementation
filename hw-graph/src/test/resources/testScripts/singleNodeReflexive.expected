# Tests reflexive edge addition with a single node

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1

# Add reflexive edge
added edge e1 from n1 to n1 in graph1

# See if edge is reflexive
the children of n1 in graph1 are: n1(e1)